include/drivers/irqchip.h

```diff
@@ -9,8 +9,12 @@
 #include <dm.h>
 #include <stdint.h>
 
-#define IRQCHIP_OPS(ops) ((struct irqchip_driver_ops *)(ops))
+#define IRQCHIP_OPS(dev) ((struct irqchip_driver_ops *)((dev)->drv->ops))
 
+/**
+ * A function that can be called to handle an IRQ. The device that triggered
+ * the interrupt will be passed as the parameter.
```

Comment:
Are we not following the Doxygen guide for documenting function parameters?

Reply:
We are. I'll update those.